babel/src/board.js

```diff
@@ -35,6 +31,10 @@ export class Board {
         return triples.combine(this.triples);
     }
     
+    get atticOrbs() {
+        return _.cloneDeep(new Board(this.width, this.height, this.atticTypes).orbs);
+    }
```

Comment:
I was thinking about this. How would you change the types without specifying yet another parameter? This may be simplified further by just exposing an attic board.

That way, your attic would have types (that could be re-generated), and another attic board on top of it.

```js
return new Board(this.width, this.height)
```

All attics have to be the same width and height, no? Otherwise, the drop in stuff won't work right.